executor/executor_test.go

```diff
@@ -43,6 +47,52 @@ func (s *testSuite) TearDownSuite(c *C) {
 	s.store.Close()
 }
 
+func (s *testSuite) TestAdmin(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists admin_test")
+	tk.MustExec("create table admin_test (c1 int, c2 int, c3 int default 1, index (c1))")
+	tk.MustExec("insert admin_test (c1) values (1),(2),(NULL)")
+	r, err := tk.Exec("admin show ddl")
+	c.Assert(err, IsNil)
+	row, err := r.Next()
+	c.Assert(err, IsNil)
+	c.Assert(row.Data, HasLen, 3)
+	txn, err := s.store.Begin()
+	c.Assert(err, IsNil)
+	ddlInfo, err := inspectkv.GetDDLInfo(txn)
+	c.Assert(err, IsNil)
+	c.Assert(row.Data[0], Equals, ddlInfo.SchemaVer)
+	c.Assert(row.Data[1], DeepEquals, ddlInfo.Owner.String())
+	c.Assert(row.Data[2], DeepEquals, "")
+	row, err = r.Next()
+	c.Assert(err, IsNil)
+	c.Assert(row, IsNil)
+
+	tk.MustExec("create table admin_test1 (c1 int, c2 int default 1, index (c1))")
+	// check table test
+	r, err = tk.Exec("admin check table admin_test, admin_test1")
```

Comment:
I think we should insert some records to admin_test1, otherwise it is an empty table.